gossip/node.go

```diff
@@ -374,7 +381,7 @@ func (n *Node) handleSnowballerDone(msg *snowballerDone) {
 		n.logger.Errorf("error publishing current round: %v", err)
 	}
 
-	round := newRound(completedRound.height + 1)
+	round := newRound(completedRound.height+1, 0, 0, min(defaultK, int(n.notaryGroup.Size())-1))
```

Comment:
If this improves things, might be good to move the last arg's code into a `func k(ng *types.NotaryGroup) int` or similar and use that above too. This is fine for now though.